Fix mini typos in docs (#958)
alan-turing-institute/MLJ.jl: docs/src/controlling_iterative_models.md

estimates.

Some iterative model implementations enable some form of automated

> control, with the method and options for doing so vary from model

Reviewer: ```suggestion
control, with the method and options for doing so varying from model
```

[`WithMachineDo`](@ref MLJIteration.WithMachineDo)`(f->mach->@info("report: $mach))`| Call `f(mach)` wher `mach` is the training machine in its current state    | yes
[`Save`](@ref MLJIteration.Save)`(filename="machine.jls")`|Save current training machine to `machine1.jls`, `machine2.jsl`, etc                         | yes

> > Table 1. Atomic controls. Some advanced options were omitted.

Reviewer: ```suggestion
> Table 1. Atomic controls. Some advanced options are omitted.
```

[IterationControl.jl](https://github.com/ablaom/IterationControl.jl). Rather
than iterating a training machine directly, we iterate a wrapped

> version of this object, which includes other information that controls

Reviewer: ```suggestion
version of this object, which includes other information that a control
```
